scripts/kcc

```diff
@@ -41,6 +41,16 @@ if ($cygwin) {
   chop($distDirectory);
   chop($profileDirectory);
 }
+
+my $preprocessor;
+{
+  local $/=undef;
+  open FILE, "$profileDirectory/pp" or die "Couldn't find current profile: run kcc-switch";
+  $preprocessor = <FILE>;
+  close FILE;
+  chop($preprocessor);
+}
+
```

Comment:
Did you consider maybe just making $profileDirectory/pp the executable preprocessor (a script or even a link)? 

Looks good to me, though.